test/Makefile

```diff
@@ -33,6 +34,16 @@ LDLIBS=\
     -lboost_program_options -lboost_system -lboost_thread -lboost_filesystem \
     -lpthread -lyajl -lssl -lcrypto -lyaml-cpp
 
+UNAME_S=$(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
+  CPPFLAGS+= -UNOOPT -DNOOPT='[[clang::optnone]]' -I/usr/local/include
+  CXXFLAGS+= -Wno-deprecated-declarations -Wno-c++14-extensions
+  LDFLAGS+= -L/usr/local/lib
+  SED_I+= ''
```

Comment:
I might be confused here, but it doesn't look like SED_I or SED_EXTRA are used below in this file; do we need these lines?

Reply:
They are not, but this is a verbatim copy from `src/Makefile` — I think it's good to keep them in sync so that we can factor this out.

Reply:
Turns out you were right — these weren't needed. Removed.